src/bosh-google-cpi/integration/stemcell_test.go

```diff
@@ -0,0 +1,44 @@
+package integration
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Stemcell", func() {
+
+	BeforeEach(func() {
+		Expect(googleProject).ToNot(Equal(""), "GOOGLE_PROJECT must be set")
+		Expect(externalStaticIP).ToNot(Equal(""), "EXTERNAL_STATIC_IP must be set")
+	})
+
+	It("can create a stemcell from a valid JSON request", func() {
+
+		request := fmt.Sprintf(`{
+			  "method": "create_stemcell",
+			  "arguments": ["", {
+				  "name": "bosh-google-kvm-ubuntu-trusty",
+				  "version": "3215",
+				  "infrastructure": "google",
+				  "source_url": "%s"
+				}]
+			}`, stemcellURL)
+
+		stemcellCID = assertSucceedsWithResult(request).(string)
```

Comment:
Possible optimization for later, but typically we run our ginkgo tests in CI in parallel + random order with `ginkgo -p --randomizeAllSpecs --randomizeSuites`. This speeds things up and avoids test pollution causing false positives.

You could combine these two `It` blocks and add in cosmetic `By`s to keep the test readable and self-contained:
```
It("executes the stemcell lifecycle", func() {
  By("uploading a stemcell")
  // run upload command

  By("deleting the stemcell")
  // run delete command
})
```